hicommonwealth/commonwealth
# Added token trading widget inside community sidebar

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/9489

## Description of Changes
Added token trading widget inside the community sidebar

## "How We Fixed It"
N/A

## Test Plan
1. Visit any community that has a token
2. Verify you see the community token loaded as a widget in the community sidebar
3. Verify you are able to trade (buy/sell) token from that widget
4. Verify that token pricing data and market caps displayed are correct

## Deployment Plan
N/A

## Other Considerations
N/A
